build_scripts/build_runner.py

```diff
@@ -179,9 +179,23 @@ def run(self):
 
                 self.log.debug(completed_process.stdout)
             except subprocess.CalledProcessError as process_error:
-                self.log.error(process_error.stdout)
+                if self.stage == Stage.BUILD:
+                    self._parse_logs(process_error.stdout)
+                else:
+                    self.log.error(process_error.stdout)
                 raise
 
+    def _parse_logs(self, stdout):
+        self.log.debug(stdout)
+        output = ""
+
+        substring = ' error ' if platform.system() == 'Windows' else ' error:'
+        for string in stdout.split('\n'):
+            if substring in string:
+                output += '\n' + string
```

Comment:
I think it is better to use list for the variable `output` and .append() each element after that you can:
```
self.log.error("\n".join(output))
```
So you will reduce number of sting instances.

Reply:
+100500

Reply:
How will we add '\n' if output contains only one element?

Reply:
I think, that you can initialize output with first empty element "": `output = [""]`  
In this case you will have additional `\n` all the time :)